content/news/2024-10-30-maplibre-newsletter-october-2024/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "MapLibre Newsletter October 2024"
+date: "2024-11-03"
+categories: ["newsletter"]
+authors: [meena]
+draft: false
+---
+
+# MapLibre Newsletter - November 2024
+
+We’re delighted to begin the November newsletter with a gracious and hearty Thank You to Meta for their unwavering support as a sponsor of MapLibre in 2024! We look forward to what is to come.
+
+<a href="https://www.meta.com/">
+<img src="Meta.png" width="100%"/>
+</a>
+<br />
+<br />
+
+## MapLibre GL JS
+
+We have released versions 5.0.0-pre.2, 5.0.0-pre.3 and 5.0.0-pre.4 this month.
+One interesting feature that was recently added was the support for pitch larger than 90 degrees.
+
+Here's an interesting situation that should be possible now:
+
+<a href="pitchlarger90degrees.png">
+<img src="pitchlarger90degrees.png" width="100%"/>
+</a>
+<br/>
+<br/>
+
+An interesting read on how this works can be found [here] (https://github.com/maplibre/maplibre-gl-js/blob/49d7fc96ad883b4e1042fe305cd0c0a16ccd9732/developer-guides/center-point.md)
```

Comment:
This Markdown link is broken.

See Markdown syntax here: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax